packages/next-ui/ActionCard/ActionCardListForm.tsx

```diff
@@ -51,29 +56,53 @@ export function ActionCardListForm<
       defaultValue={defaultValue}
       rules={{ required: errorMessage || required, ...rules }}
       render={({ field: { onChange, value, ref }, fieldState, formState }) => (
-        <ActionCardList
-          {...other}
-          multiple={multiple}
-          required={required}
-          value={value}
-          ref={ref}
-          onChange={(_, incomming) => onChange(incomming)}
-          error={formState.isSubmitted && !!fieldState.error}
-          errorMessage={fieldState.error?.message}
-        >
-          {items.map((item) => (
-            <RenderItem
-              {...item}
-              key={item.value ?? 'tralala'}
-              value={item.value}
-              selected={onSelect(item.value, value)}
-              onReset={(e) => {
-                e.preventDefault()
-                onChange(null)
+        <>
+          <ActionCardList
+            {...other}
+            multiple={multiple}
+            required={required}
+            value={value}
+            ref={ref}
+            onChange={(_, incomming) => onChange(incomming)}
+            error={formState.isSubmitted && !!fieldState.error}
+            errorMessage={fieldState.error?.message}
+          >
+            {items.map((item, index) => (
+              <RenderItem
+                {...item}
+                key={item.value ?? ''}
+                value={item.value}
+                selected={onSelect(item.value, value)}
+                onReset={(e) => {
+                  e.preventDefault()
+                  onChange(null)
+                }}
+                index={index}
+                show={show}
+                maxItems={maxItems}
+              />
+            ))}
+          </ActionCardList>
+          {items.length > maxItems && (
+            <Button
+              sx={{
+                my: 2,
               }}
-            />
-          ))}
-        </ActionCardList>
+              color='primary'
+              variant='text'
+              onClick={() => setShow(!show)}
+            >
+              {!show ? <Trans id='More options' /> : <Trans id='Less options' />}{' '}
+              <IconSvg
+                sx={{
+                  transform: show ? 'rotate(180deg)' : 'rotate(0deg)',
+                  transition: 'transform 0.3s ease-in-out',
+                }}
+                src={iconChevronDown}
+              />
+            </Button>
+          )}
+        </>
```

Comment:
I'm going to be annoying here, what if the currently selected item is in the list you've collapsed, shouldn't it still be shown?